indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -0,0 +1,138 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import dashboardURL from 'indico-url:event_editing.dashboard';
+// import manageEditableTypeListURL from 'indico-url:event_editing.manage_editable_type_list';
+
+import React, {useState} from 'react';
+import {useParams} from 'react-router-dom';
+import {Button, Icon, Checkbox} from 'semantic-ui-react';
+import {Column, Table, SortDirection, WindowScroller} from 'react-virtualized';
+import _ from 'lodash';
+import {
+  TooltipIfTruncated,
+  ManagementPageSubTitle,
+  ManagementPageBackButton,
+} from 'indico/react/components';
+import {useNumericParam} from 'indico/react/util/routing';
+import {Translate} from 'indico/react/i18n';
+import {EditableTypeTitles} from '../../models';
+import StateIndicator from '../../editing/timeline/StateIndicator';
+import './EditableList.module.scss';
+
+import list from './list'; // will remove
+
+export default function EditableList() {
+  const eventId = useNumericParam('confId');
+  const {type} = useParams();
+  const [sortBy, setSortBy] = useState('id');
+  const [sortDirection, setSortDirection] = useState('ASC');
+  const [sortedList, setSortedList] = useState(list);
+
+  // eslint-disable-next-line no-shadow
+  function _sortList({sortBy, sortDirection}) {
+    const newList = _.sortBy(list, [sortBy]);
+    if (sortDirection === SortDirection.DESC) {
+      newList.reverse();
+    }
+    return newList;
+  }
+
+  // eslint-disable-next-line no-shadow
+  function _sort({sortBy, sortDirection}) {
+    setSortBy(sortBy);
+    setSortDirection(sortDirection);
+    setSortedList(_sortList({sortBy, sortDirection}));
+  }
+
+  function _cellRenderer(dataKey, rowIndex) {
+    return (
+      <>
+        {dataKey === 'status' && <StateIndicator index={rowIndex} state="accepted" circular />}
+        <TooltipIfTruncated>
+          <div index={rowIndex} className="rowColumn_tooltip" role="gridcell">
+            {sortedList[rowIndex][dataKey]}
+          </div>
+        </TooltipIfTruncated>
+      </>
+    );
+  }
+
+  return (
+    <>
+      <ManagementPageSubTitle title={`List of ${EditableTypeTitles[type]}s`} />
```

Comment:
missing i18n (needs 3 separate strings since interpolation is not good in translatable strings)